tests/test_shared_id.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from _helpers import *
+
+
+@pytest.mark.asyncio
+async def test_sharedId_in_same_realm_same_navigable(websocket, context_id):
+    await goto_url(
+        websocket, context_id,
+        "data:text/html,<div some_attr_name='some_attr_value' "
```

Comment:
nit/optional: we could use `"""` strings to make this HTML block nicely formatted (in multiple lines)

Reply:
replaced with a simpler page